Feature/model

KorPaEv opened the pull request:
Add Models

KorPaEv committed:
Create DEBET-CREDIT Model class

KorPaEv committed:
Create Operation Type ENUM CONST for Decred class

KorPaEv committed:
Add OperaionType column

KorPaEv committed:
Edit Description

KorPaEv committed:
Rename class
Add creationRecordDateTime column

KorPaEv committed:
Add User model Class

KorPaEv committed:
Add DB connection

KorPaEv committed:
Add Entity Relationships

KorPaEv committed:
Change oneToMany on List

palestiner reviewed (changes requested):
Убрать из конструкторов всех entity их id
DecredItem.java:
1) Правильно настроить зависимость для paymentCategory
2) Настроить operationType через Converter (https://thorben-janssen.com/hibernate-enum-mappings/)
3) Переписать equals через instanceof
4) Переписать hashCode

в application-dev.yaml записать с дефолтным значением username и password:
- ${DECRED_DB_USERNAME:postgres}
- ${DECRED_DB_PASSWORD:qaz123321999}

перенести spring.jpa.show-sql: true в application-dev

KorPaEv committed:
Change default values

KorPaEv committed:
Refactoring
- User class
- DecredItem class
- PaymentCategory class
- OperationType

Create OperationTypeConverter class
